### DISCUSSION TITLE
API/SDK to create comments on traces

### DISCUSSION THREAD

**Discussion by darvin** [langfuse/langfuse#3263] [Ideas]
### Describe the feature or potential improvement

it would be nice to have api for comments

**Comment by clemra** [comment]
Coming soon :) tagging @marliessophie here. We'll update you here.

**Reply by jcbmllgn** [reply]
Hey @clemra , is there an updated ETA for this feature? thanks!

**Comment by jannikmaierhoefer** [comment]
@jcbmllgn @darvin this is now available via the API and the SDK: https://api.reference.langfuse.com/#tag/comments

**Reply by jcbmllgn** [reply]
great! Thanks.
